### ISSUE TITLE
Create Markings Work Scheduling page in SMB Tracker

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#1332]
Esteban G.
SMB

He called asking for help getting access to SharePoint. Apparently Christina has made a calendar for documenting work scheduling in SMB per Jim D. I saw that he'd have to copy/paste 3 weeks of marking work order information into the calendar. I mentioned there is definitely a way to do it in Knack where he wouldn't have to copy/paste all the information.

**Comment by dianamartin** [comment]
I mocked up the page and called Christina the next day to confirm if they really wanted to pursue the SharePoint calendar. She said that doing that way would be painful for the markings supervisors, but that Jim D. said he didn't want to wait to update Data Tracker. 

I screen shared with Christina to set up the page so she could show it to Esteban and get his feedback on the page.

**Comment by dianamartin** [comment]
She talked with him and emailed me functionality I should refine: 

>Thanks for your help this morning. I reviewed the Work Scheduling Calendar and these are a couple changes Esteban and I were thinking:

### Unscheduled Jobs
- [ ] Add “Created date”
- [ ] Hyperlink on "Requester ID" field to link to work order
- [ ] Group jobs based on “Requester ID”

### Scheduled Jobs

- [ ] Need ability to edit scheduled work dates
- [ ] Fields to match unscheduled work fields

### Calendar:

- [ ] Color based on work group
- [ ] Have "Requester ID" and "Location" on calendar event

>Esteban is going to look at it more and see if there is anything else needed, but we are SUPER excited about this. 😄

**Comment by dianamartin** [comment]
- [ ] Display Name 
- [ ] Viewer calendar (work-schedule) - no filter 
- [ ] Technician calendar (work-schedule) 
- [ ] colors for work groups: manage schedule, technician

**Comment by dianamartin** [comment]
Messaged Christina: 
>So I added the "Work Schedule" (Viewing) on the Markings Work Order page, next to the "New Work Order" button. On the "Work Schedule" page, I added a new menu for "Manage Schedule" (for supervisors only) and also added it on the bottom admin menu as well in case you need to get to it for some reason.

**Comment by dianamartin** [comment]
Updated the CSS code to have the filter colors on the calendars highlight. Need to add it to the code repo.

**Comment by dianamartin** [comment]
Need to ask @SurbhiBakshi about my "display name" error on Monday.

**Comment by dianamartin** [comment]
Christina was able to review and is happy with the page. She was going to show Esteban today once he gets back from the field.

**Comment by dianamartin** [comment]
Emailed Christina

>Just checking in to see if Esteban had any feedback or had any changes he wanted to make on this?

**Comment by dianamartin** [comment]
Christina emailed on 2/20 saying that Esteban had a few requested changes: 
>Esteban has been using the markings work scheduling and has comments of things he would like to modify:
- [x] Add “Coordination” field to the “Unscheduled Markings Jobs” and “Scheduled Markings Work” tables
  - Added the `Coordination` field to the Unscheduled and Scheduled markings jobs tables and to the event details pop-up
- [x] Needs a way to create events to block out days that are training days and bad weather days. When we first worked on this, I remember there was a way to create events on the calendar so this function should work for Esteban. 
  - Tested out creating a job by using the `Job Location Description` field (short text) and is used for one very particular reason. We were able to make it look okay on the calendars (Manage Schedule, Work Schedule - Tech, Work Schedule - Viewer)
- [x] Needs the ability to schedule 1 job multiple times. Example: some work order take several weeks and are not worked on consecutively.
  - Asked Surbhi about this, looks like the repeating calendar function might work for Esteban since he doesn't need to create a new job record

Might need to work on the filter for the Unscheduled and Schedule Jobs table today (2/21)
